public/javascripts/tool_inline.js

```diff
@@ -137,61 +138,4 @@ $(function() {
   })
 });
 
-window.addEventListener('message', function(e) {
-  try {
-    var message = JSON.parse(e.data);
-    switch (message.subject) {
-      case 'lti.frameResize':
-        const toolResizer = new ToolLaunchResizer();
-        var height = message.height;
-        if (height <= 0) height = 1;
-
-        const container = toolResizer.tool_content_wrapper(message.token || e.origin).data('height_overridden', true);
-        toolResizer.resize_tool_content_wrapper(height, container);
-        break;
-
-      case 'lti.showModuleNavigation':
```

Comment:
I appreciate the cleanup of extracting all of the post message code to a separate file. But since all of that code was touched, we should add additional tests for all the code extracted

Reply:
I've added the additional specs as requested and they pass when I run them. Let me know if you there is anything else you need. Thanks!